frontend/src/views/account-mgmt/ManageOrganizationView.vue

```diff
@@ -180,6 +194,20 @@ export default {
       this.showChangeRequestDialog = !this.showChangeRequestDialog
     },
 
+    async validateOfmProgram() {
+      try {
+        this.loading = true
+        if (!this.hasAnOFMFacility && ! await ApplicationService.hasActiveApplicationOrFundingAgreement(this.userInfo.facilities)) {
```

Comment:
It's probably only me, but I've never seen "! await" in if/else condition. I think it is easier to read if we have it in a separate line and maybe assign a variable to it. (this is just a comment. it is your call!)

Reply:
Changed as suggested. I've only recently seen that style watching someone's code in a youtube channel ;)... i like it since less code, however you're a more experienced javascript dev so i value your preferences. :)

Reply:
nah.. I'm a newbie that's why I've never seen it before. also, your initial approach is more efficient because we won't call "hasActiveApplicationOrFundingAgreement" unless it has no OFM facility. If that "hasActiveApplicationOrFundingAgreement" request could take long time, we can also move that request inside the if condition. but, it's again your call... I'm fine with what we have.